### PR TITLE
♿ [amp-consent] fix: amp-consent iframe title for accessibility

### PR DESCRIPTION
**Pull request by honzalo**
Addresses the title name of the iframe inside the `<amp-consent>` element.
Initial MR https://github.com/ampproject/amphtml/pull/39940 but this was targeting the wrong iframe.

### CODE REVIEW

File: extensions/amp-consent/0.1/amp-consent.js [hunk 1]
```diff
@@ -1035,7 +1035,10 @@ export class AmpConsent extends AMP.BaseElement {
 
     // Set up the __tcfApiLocator window to signal postMessage support
     const iframe = this.element.ownerDocument.createElement('iframe');
-    iframe.setAttribute('title', this.element.title || 'Consent Banner');
+    iframe.setAttribute(
```
Comment (powerivq) [inline extensions/amp-consent/0.1/amp-consent.js:1038]: Should this be removed, if this iframe is not useful to the user?

File: extensions/amp-consent/0.1/consent-ui.js [hunk 2]
```diff
@@ -471,6 +471,7 @@ export class ConsentUI {
    */
   createPromptIframe_(promptUISrc) {
     const iframe = this.parent_.ownerDocument.createElement('iframe');
+    iframe.setAttribute('title', this.parent_.title || 'Consent Banner');
```
Comment (powerivq) [inline extensions/amp-consent/0.1/consent-ui.js:474]: Nit: move this one row below.

**Review by powerivq — CHANGES_REQUESTED** [review]

File: extensions/amp-consent/0.1/consent-ui.js (hunk 2 above)
Comment (honzalo) [reply extensions/amp-consent/0.1/consent-ui.js]: Thanks, updated with daab62b58aa12b457260647c64dda45d21bc0dd9

File: extensions/amp-consent/0.1/amp-consent.js (hunk 1 above)
Comment (honzalo) [reply extensions/amp-consent/0.1/amp-consent.js:1038]: I'm not sure the iframe is not useful for the user. The iframe is used for setting up the __tcfApiLocator so this change is only trying to reflect that correctly in the iframe title, as the Consent Banner iframe is really the one setup in the consent-ui file.

Comment (powerivq) [reply extensions/amp-consent/0.1/amp-consent.js:1038]: Sorry. I meant that this iframe is not user-accessible, and therefore it serves no purposes to set its title attribute. We should simply revert https://github.com/ampproject/amphtml/commit/5e5106bdcf2db0656b86539252f6d4f5ffc533be by removing this row altogether.

**Review by powerivq — CHANGES_REQUESTED** [review]

File: extensions/amp-consent/0.1/amp-consent.js (hunk 1 above)
Comment (honzalo) [reply extensions/amp-consent/0.1/amp-consent.js]: Sorry, I miss interpreted your comment. Yes, that makes sense, great catch. I've removed it with 53fa8ba5dfd4acf0aee3be2694edf974334dfd4e. Thanks!

**Review by powerivq — APPROVED** [review]

**Comment by powerivq** [comment]
@honzalo Can you help rebasing against latest main branch? It would probably fix the test failure.

**Pull request merged**